src/VisualStudio.Roslyn.SDK/Roslyn.SDK.Template.Wizard/RoslynSDKRootTemplateWizard.cs

```diff
@@ -13,7 +13,7 @@ public partial class RoslynSDKRootTemplateWizard
 <configuration>
   <packageSources>
     <clear />
-    <add key=""nuget.org"" value=""https://api.nuget.org/v3/index.json"" />
+    <add key=""dotnet-public"" value=""https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json"" />
```

Comment:
@mmitche This one is a public template for third-party extenders, and should not be updated. How do I fix it such that it doesn't trigger automation?

Reply:
If it's a third party template, does it need our pre-shipping tools feed? Could it just be nuget.org?

Reply:
@mmitche The Microsoft.CodeAnalysis.Testing package aren't on nuget.org yet. I personally feel it should be in nuget.org and eliminate nuget.org from the template (or only use nuget.org from it)

Reply:
Ideally we'd only use nuget.org for this. When will these packages go to nuget.org?

Reply:
> When will these packages go to nuget.org?

@sharwell @jmarolf Any plans?

Reply:
This also does seem to block dotnet/docs :confused:

https://github.com/dotnet/docs/pull/23231#discussion_r590601930

cc: @gewarren

Reply:
Are these packages likely to move to nuget? If not, we'll probably have to remove Youssef's sample from dotnet/docs and dotnet/samples repos. The vulnerability in the nuget.config file blocks our publishing pipelines in those two repos.

Reply:
@gewarren Note that in its current form, the nuget.config is **not** vulnerable. Would you be able to contact the team who owns the tooling that do this check to recognize azure feeds as private, and not flagging a nuget.config if **all** feeds are azure feeds?

The discussion above is more about, "I'm a consumer that wants to **extend** the sample and use a package from nuget.org - in this case it *could* be vulnerable"

Reply:
It turns out there's a variable you can add to the pipeline so it doesn't fail the build. So we're all set now.